src/project_parser.cpp

```diff
@@ -895,5 +898,17 @@ bool is_root_path(const std::string &path) {
     return toml.contains("project");
 }
 
+bool Project::is_valid_hex(std::string value){
```

Comment:
Since this function is very trivial and only used once I would put the code directly at the check.

If not, please put the function as a static function directly in project.cpp